townsfolk/power/hooker

```diff
@@ -1,6 +1,6 @@
 **Hooker** | Townsfolk Power
 __Basics__
-Each night, the Hooker sleeps with another player. They will not be killed by attacks, but will be affected by those taking place at their target’s house.
+Each night, the Hooker sleeps with another player. They will not be killed by attacks on them, but will be affected by those taking place at their target’s house.
 __Details__
-Since the Hooker is at the other player’s house and not at home, any attacks in the night will fail; however, if the person they’re sleeping over at is being attacked, then so are they.
+Since the Hooker is at the other player’s house and not at home, any attacks targeting the Hooker in the night will fail; however, if the person they’re sleeping over at is being attacked, then so are they.
```

Comment:
```suggestion
Since the Hooker is at the other player’s house and not at home, any attacks targeting the Hooker in the night will fail; however, if the person they’re sleeping over at is being attacked, then they both are attacked.
```